googlemock/include/gmock/gmock-matchers.h

```diff
@@ -1599,7 +1599,10 @@ struct MatcherList<2, Matcher1, Matcher2> {
         SafeMatcherCast<T>(matchers.second)));
   }
 };
-
+// Skip variadic implementation of matchers if using GCC < 4.7 due to
+// Bug 35722 -[C++0x] Variadic templates expansion into non-variadic class template
+// https://gcc.gnu.org/bugzilla/show_bug.cgi?id=35722
+#if !defined(__GNUC__) || (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 7 ) )
```

Comment:
Let's make a macro for the feature in the -port.h instead of chopping up the compiler version inline with the code.